cpp/Platform.Timestamps.Tests/CMakeLists.txt

```diff
@@ -26,6 +26,6 @@ if(NOT googletest_POPULATED)
     add_subdirectory(${googletest_SOURCE_DIR} ${googletest_BINARY_DIR})
 endif()
 
-add_executable(${PROJECT_NAME}.Tests ${PROJECT_NAME}.Tests.cpp)
+add_executable(${PROJECT_NAME}.Tests CommonEraClockTests.cpp UniqueTimestampFactoryTests.cpp)
```

Comment:
Use 
```cmake
add_executable(${PROJECT_NAME}.Tests ${PROJECT_NAME}.Tests.cpp)
```
```cpp
// in the ${PROJECT_NAME}.Tests.cpp
#include "CommonEraClockTests.cpp"
#include "UniqueTimestampFactoryTests.cpp"
```
OR 
```cmake
file(GLOB SRC_FILES *.cpp)
add_executable(${PROJECT_NAME}.Tests ${SRC_FILES})
```

Reply:
> `// in the ${PROJECT_NAME}.Tests.cpp`
> `#include "CommonEraClockTests.cpp"`
> `#include "UniqueTimestampFactoryTests.cpp"`

I had already done this, but it's not compatible with the cpp-cs translator style that we're using. The second option seems really good though